mysql-test/suite/innodb_fts/t/percona_bug_ps3928.test

```diff
@@ -0,0 +1,49 @@
+# Bug PS-3928: Complex case insensitive full text queries returned no results
```

Comment:
Use upstream bug id for testcase naming and in the comments inside (see Confluence)